ntbs-service/Pages/Admin/Migration.cshtml

```diff
@@ -1,105 +1,104 @@
-                    <nhs-date-input-item>
-                        <nhs-form-group nhs-form-group-type="Standard">
-                            <label nhs-label-type="Date" asp-for="NotificationDateRangeEnd.Day">Day</label>
-                            <input nhs-input-type="Date" fixed-width="Two"
-                                is-error-input=@notificationDateEndError asp-for="NotificationDateRangeEnd.Day" />
-                        </nhs-form-group>
-                    </nhs-date-input-item>
-                    <nhs-date-input-item>
-                        <nhs-form-group nhs-form-group-type="Standard">
-                            <label nhs-label-type="Date" asp-for="NotificationDateRangeEnd.Month">Month</label>
-                            <input nhs-input-type="Date" fixed-width="Two"
-                                is-error-input=@notificationDateEndError asp-for="NotificationDateRangeEnd.Month" />
-                        </nhs-form-group>
-                    </nhs-date-input-item>
-                    <nhs-date-input-item>
-                        <nhs-form-group nhs-form-group-type="Standard">
-                            <label nhs-label-type="Date" asp-for="NotificationDateRangeEnd.Year">Year</label>
-                            <input nhs-input-type="Date" fixed-width="Four"
-                                is-error-input=@notificationDateEndError asp-for="NotificationDateRangeEnd.Year" />
-                        </nhs-form-group>
-                    </nhs-date-input-item>
-                </nhs-date-input>
-            </nhs-form-group>
-        </div>
-
-        <div>
-            <button nhs-button-type="Standard" type="submit" value="Import" classes="ntbsuk-button--primary submit-search-button" id="search-button">
-                Import
-            </button>
-        </div>
-    </form>
-
-    <br/>
-    <a href='/Hangfire/jobs/succeeded'> Navigate to dashboard </a>
+@page
+@model ntbs_service.Pages.Admin.MigrationModel
+@using static NHSUK.FrontEndLibrary.TagHelpers.FormGroupType
+
+@{
+    ViewData["Title"] = "Migration";
+    Layout = "Shared/_Layout";
+}
+
+<nhs-width-container container-width="Standard">
+    <h1>Import Legacy Notifications</h1>
+
+    <form method="post" enctype="multipart/form-data">
+        <input nhs-input-type="Standard" asp-for="NotificationIds">
+        <br/>
+
+        <div>
+            <nhs-form-group>
+                <label asp-for="UploadedFile">Upload File</label>
+                <input type="file" asp-for="UploadedFile"/>
+            </nhs-form-group>
+        </div>
+        <br/>
+
+        <div>
+            @{
+                var notificationDateStartError = !Model.ValidationService.IsValid("NotificationDateRangeStart");
+                var notificationDateStartGroupState = notificationDateStartError ? Error : Standard;
+            }
+            <nhs-form-group nhs-form-group-type=@notificationDateStartGroupState classes="nhs-form-group--import-by-date">
```

Comment:
I believe rider should complain about the lack of quotes around `@notificationDateStartGroupState` here - I'm not convinced it has any real reason to... but I've been fixing it in other places.

Reply:
(for me at least) it just offers the change, but without highlighting it as an error or even warning. I think it's fine this way, it's clear enough and more concise, but i could easily be convinced otherwise :P